seed/data_importer/meters_parser.py

```diff
@@ -378,7 +356,14 @@ def _parse_cost_meter_reading(self, raw_details, meter_details, start_time, end_
             'conversion_factor': 1
         }
 
-        # Associate this meter reading to each meter and property_id combination
+        self.distribute_meter_reading(meter_reading, meter_details)
+
+    def distribute_meter_reading(self, meter_reading, meter_details):
+        """
+        If a meter has not yet been parsed, its first reading details are saved
+        in a list. If a meter was previously parsed and has readings already,
+        any new readings are appended to that list.
+        """
         for property_id in meter_details.get('property_ids', []):
```

Comment:
The diff isn't showing as I expected, but the DRY'ed code can be found in this new method.